connector/connect/client/jvm-internal/src/main/scala/org/apache/spark/sql/connect/client/GrpcExceptionConverter.scala

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.spark.sql.connect.client
 
-import scala.jdk.CollectionConverters._
+import scala.collection.JavaConverters._
```

Comment:
lint build complained for some reason

Reply:
hmm... `scala-collection-compat_2.12-2.8.1.jar` also provides `scala.jdk.CollectionConverters`

Reply:
I guess the jvm-internal module must be missing dependency on `scala-collection-compat_2.12-2.8.1.jar` somehow.
I don't see it explicit in any POM, so I guess it gets pulled trasitively by something in some of the modules with more deps.